meerk40t/main.py

```diff
@@ -349,6 +355,8 @@ def run():
                 kernel_root(line.strip() + "\n")
         kernel_root.channel("console").unwatch(print)
 
+    kernel.bootstrap("ready")
```

Comment:
This has functional impact. The ready lifecycle event in 0.7.x version does a flag for "on_controller_ready" for the lhystudios devices this causes the controller thread in the lhystudios driver and the moshiboard driver to kick in. This is also when the start driver for the drivers kicks in which does driver threading. This is where it starts the fetch/execute loop for the spooler. and this also is when the tree is initially built.

The consequence here is that batch and execute will happen before these events.  I believe this ordering is because "quit" is a valid command and was a common thing for -q for a while. This might be an acceptable ordering but the `quit` setting would typically work when the fetch execute loop finds an empty spooler. In this ordering that really could happen even with `execute` and `batch` commands.

At a minimum this change should *not* be part of improving crash reporting and should be justified in an independent PR. (also the change would be obsoleted by 0.8.0).

Reply:
Ok - I can put this back and add a new `bootstrap("finished")` event for any plugins that want to run stuff after boot has finished.